v20.2/experimental-features.md

```diff
@@ -19,6 +19,7 @@ The table below lists the experimental session settings that are available.  For
 
 | Variable                            | Default Value | Description                                                                                                                                                                                                                                                                                             |
 |-------------------------------------+---------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `enable_experimental_alter_column_type_general`       | `'false'`       |  If set to `'true'`, enables [column type altering](#alter-column-type).                                                                                                                                                                                   |
```

Comment:
I think it would be useful to specify it enables ALTER COLUMN TYPE support for general cases with some limitations or something along those lines.

Also I don't think the hyperlink is working properly here.